runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -166,13 +171,15 @@ public void onChatMessage(ChatMessage event)
 				//set it to 17 because this message is triggered first before the above chat event
 				bindNeckOverlay.bindingCharges = 17;
 			}
+			return;
```

Comment:
This should be within the above `if (match.find())` block, after the line where `bindNeckOverlay.bindingCharges` is set.